File: Level1/행렬의 덧셈.cpp
#include <string>
#include <vector>

using namespace std;

vector<vector<int>> solution(vector<vector<int>> arr1, vector<vector<int>> arr2) {
    vector<vector<int>> answer;
    for (int r = 0; r < arr1.size(); r++) {
        vector<int> tmp;
        for (int c = 0; c < arr2[0].size(); c++) {
            int value = arr1[r][c] + arr2[r][c];
            tmp.push_back(value);
        }
        answer.push_back(tmp);
    }
    return answer;
}

File: Level1/행렬의 덧셈.py
def solution(arr1, arr2):
    #정답 벡터
    answer = [[] for _ in range(len(arr1))]
    
    #하나씩 더해줌
    for i in range(len(arr1)):
        for j in range(len(arr1[0])):
            answer[i].append(arr1[i][j]+arr2[i][j])
    
    #반환
    return answer